utils/sv_IO_panel_tools.py

```diff
@@ -145,7 +142,41 @@ def get_superficial_props(node_dict, node):
     node_dict['label'] = node.label
     node_dict['hide'] = node.hide
     node_dict['location'] = node.location[:]
-    node_dict['color'] = node.color[:]
+    if node.use_custom_color:
+        node_dict['color'] = node.color[:]
+        node_dict['use_custom_color'] = True
+
+
+def collect_custom_socket_properties(node, node_dict):
+    print("** PROCESSING custom properties for node: ", node.bl_idname)
+    input_socket_storage = { }
+    for socket in node.inputs:
+
+        # if not tracked_socket(socket): continue
+        print("Socket %d of %d" % (socket.index+1, len(node.inputs)))
+
+        storable = {}
+        tracked_props = 'use_expander', 'use_quicklink', 'expanded', 'use_prop'
```

Comment:
somewhere here you should be able to check against   the socket.<properties>.default , and store only if the current value is modfied.  like... `socket.bl_rna.properties[property_name].default == socket.property_name`  @DolphinDream

Reply:
socket.bl_rna.properties[property_name].default == getattr(socket, property_name)